application/Modules/Clipboard/Producers/ClipboardCopyEventProducer.cs

```diff
@@ -34,121 +35,22 @@ private void OnClipboardUpdate(IntPtr hWnd, uint messageId, IntPtr wParam, IntPt
-
-                INativeClipboard.WindowClass windowClass;
-
-                windowClass.style = 0;
-                windowClass.lpfnWndProc = internalWindowMessageHandler;
-                windowClass.cbClsExtra = 0;
-                windowClass.cbWndExtra = 0;
-                windowClass.hInstance = IntPtr.Zero;
-                windowClass.hIcon = IntPtr.Zero;
-                windowClass.hCursor = IntPtr.Zero;
-                windowClass.hbrBackground = IntPtr.Zero;
-                windowClass.lpszMenuName = "";
-                windowClass.lpszClassName = className;
-
-
-                nativeClipboard.RegisterClass(ref windowClass);
-
-                // Creates window to register clipboard update messages
-                WindowHandle = nativeClipboard.CreateWindowEx(0,
-                                                              className,
-                                                              "",
-                                                              0,
-                                                              0, 0,
-                                                              1, 1,
-                                                              IntPtr.Zero,
-                                                              IntPtr.Zero,
-                                                              IntPtr.Zero,
-                                                              IntPtr.Zero);
-
-                nativeClipboard.AddClipboardFormatListener(WindowHandle);
+                text = nativeClipboard.GetClipboardText();
             }
-
-            /// <summary>
-            ///     The underlying window handle
-            /// </summary>
-            private IntPtr WindowHandle { get; }
-
-            /// <summary>
-            ///     Event invoked when clipboard is updated
-            /// </summary>
-            public event ClipboardEventHandler? ClipboardUpdated;
-
-            private IntPtr OnClipboardUpdate(IntPtr hWnd, uint messageId, IntPtr wParam, IntPtr lParam)
+            catch (Exception)
             {
-                if (messageId == (int) GlobalHook.MessageType.WM_CLIPBOARDUPDATE)
-                {
-                    ClipboardUpdated?.Invoke(hWnd, messageId, wParam, lParam);
-                }
-
-                return nativeClipboard.DefWindowProc(hWnd, messageId, wParam, lParam);
-            }
-
-            #region Dispose
-
-            private bool isDisposed;
-
-            /// <summary>
-            ///     Frees all unmanaged resources
-            /// </summary>
-            public void Dispose()
-            {
-                Dispose(true);
-            }
-
-            ~ClipboardWindowMessageSink()
-            {
-                Dispose(false);
+                return;
             }
 
-            private void Dispose(bool disposing)
+            if (Convert.ToString(wParam) == "0" || Convert.ToString(wParam) == "18")
```

Comment:
Could you instead compare this with the integer values? (e.g. something like `wParam.ToInt64() == 0 || wParam.ToInt64() == 18`). Could you additionally extract the numbers to constants and give them a meaningful name?